pennylane/io/qasm_interpreter.py

```diff
@@ -596,8 +602,11 @@ def visit_quantum_measurement_statement(
             node (QuantumMeasurementStatement): the quantum measurement statement to register.
             context (Context): the current context.
         """
-        wire = self.visit(node.measure.qubit, context)
-        res = measure(context.wire_map.get(wire, wire))
+        wires = self.visit(node.measure.qubit, context)
+        if not isinstance(wires, list):
+            res = measure(context.wire_map.get(wires, wires))
+        else:
+            res = [measure(context.wire_map.get(wire, wire)) for wire in wires]
```

Comment:
I think readability would be slightly better if the condition if not a negative here:
```suggestion
        if isinstance(wires, list):
            res = [measure(context.wire_map.get(wire, wire)) for wire in wires]
        else:
            res = measure(context.wire_map.get(wires, wires))
```